### DISCUSSION TITLE
no-fetch-if logging

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#3115] [Q&A] [answered]
`no-fetch-if` logging differs from previous behaviour
**firefox 122.0.1 , ublock origin 1.55.1b28**

**website** = `imagetranslator.io`
**filters used** = 
```
imagetranslator.io##+js(no-fetch-if)
imagetranslator.io#@#+js(no-fetch-if, googlesyndication)
```
2nd filter is to disable  current fix in ublock filters, testing

**ubo logger(1.55.1b28) shows like this** 
![Screenshot (310)](https://github.com/uBlockOrigin/uBlock-issues/assets/20338483/a573ea4c-32d2-4948-b089-23788ff701c0)

**previous behaviour in ubo 1.55.0** => 
![Screenshot (306)](https://github.com/uBlockOrigin/uBlock-issues/assets/20338483/cee03bb9-6115-4343-a8c7-ac7ce50fea09)

**Comment by gorhill** [comment, marked as answer]
This will be fixed in next dev build.
